spec/statesman/adapters/active_record_queries_spec.rb

```diff
@@ -68,7 +75,7 @@ def self.initial_state
 
         it { is_expected.to include model }
         it { is_expected.not_to include other_model }
-        its(:to_sql) { is_expected.to include('most_recent') }
+        its(:to_sql) { is_expected.to include('most_recent =') }
```

Comment:
Any reason for changing these? I think I prefer them as they were - the SQL query could be `WHERE most_recent` rather than `WHERE most_recent = 't'`, so I'd rather not add the `=`.